src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Security/AuthorizationHandlers/ViewSpecificMethodologyAuthorizationHandlerTests.cs

```diff
@@ -184,42 +203,79 @@ public async Task PrereleaseViewersOnOwningPublicationsLatestReleaseCanViewMetho
 
                 await ForEachReleaseRoleAsync(async releaseRole =>
                 {
-                    var (
-                        handler,
-                        methodologyRepository,
-                        userPublicationRoleRepository,
-                        userReleaseRoleRepository
-                        ) = CreateHandlerAndDependencies();
+                    var methodologyRepository = new Mock<IMethodologyRepository>(Strict);
+                    var userPublicationRoleRepository = new Mock<IUserPublicationRoleRepository>(Strict);
+                    var userReleaseRoleRepository = new Mock<IUserReleaseRoleRepository>(Strict);
+                    var preReleaseService = new Mock<IPreReleaseService>(Strict);
 
-                    methodologyRepository.Setup(s =>
-                            s.GetOwningPublication(MethodologyVersion.MethodologyId))
-                        .ReturnsAsync(OwningPublication);
+                    var release = new Release
+                    {
+                        ReleaseName = "2000",
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        PublicationId = OwningPublication.Id,
+                    };
 
-                    userPublicationRoleRepository.SetupPublicationOwnerRoleExpectations(
-                        UserId, OwningPublication, false);
+                    var contentDbContextId = Guid.NewGuid().ToString();
+                    await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
+                    {
+                        await contentDbContext.AddRangeAsync(
+                            OwningPublication,
+                            release);
+                        await contentDbContext.SaveChangesAsync();
```

Comment:
This caught my eye because `OwningPublication` is static.
I think prior to your changes in this PR `OwningPublication` was just used for its Id and there was no database used by the test, all the repositories were mocked, and `OwningPublication` was never mutated.

I think that it was declared static for convenience to save a few lines where similar tests all needed a Publication for its Id.

With your change adding in the in-memory database combined with the change here, `OwningPublication` is mutating.

There's about 5 user roles iterated over in the for loop that surrounds this block.  If you debug then you will see on each iteration after saving changes another Release is added to the Publication's list of Releases. By the time we skip to the following unit tests, if you debug `OwningPublication` there you will see it  has 5 Releases.

Here's a screenshot of `OwningPublication` in a different unit test.

![image](https://user-images.githubusercontent.com/4147126/160807612-9c9b68bd-7153-4b4c-9af2-bf583fc4279b.png)

That could cause some unsuspecting consequences so I think it would be best to refactor this to not use a static Publication.